mantidproject/mantid
# Crystal Field physical properties

This PR adds the ability to calculate and fit physical properties (heat capacity, magnetic susceptibility and magnetisation) to the crystal field module (port of VMS Focus programs).

**To test:**

Check script here: https://www.dropbox.com/s/1u4yp6n3b7d3w6a/mantid_cef_physprop.py?dl=0 works without errors. Check that examples in documentation page: https://dl.dropboxusercontent.com/content_link/ipoOYxryHUclu0Eb4j1m1SyTohBm2MZwjAKpdSyzbBensQFPjgitOeYd13AFklke/file#calculating-physical-properties works without errors.



Fixes #17983 



Documentation updated. Release notes added.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
